Ecommerce/ecommerce_app/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Product(models.Model):
+    product_name = models.CharField(max_length = 255)
+    sku = models.CharField(max_length = 50, unique = True)
+    price = models.JSONField()
+    description = models.TextField()
+    details = models.JSONField(blank = True, null = True)
```

Comment:
would it be better to have a separate model to store details as this violates 1 normal form